guides/v2.1/cloud/env/variables-build.md

```diff
@@ -114,9 +111,9 @@ If you need to quickly debug code, use this variable to skip compilation before
 
 Set to `true` to skip static content deployment during the build phase.
 
-If you are already deploying static content during the build phase with [Configuration Management]({{ page.baseurl }}/cloud/live/sens-data-over.html), you may want to turn it off for a quick build test.
+If you already deploy static content during the build phase with [Configuration Management]({{ page.baseurl }}/cloud/live/sens-data-over.html), you may want to turn it off for a quick build test.
```

Comment:
you may want to turn it off for a quick build test.

Turn what off?   How?    do you mean set "SKIP_SCD" to false?  I'm unsure, but you know best.